people/jenna/coffeetea.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Tea v Coffee </title>
+</head>
+<body>
+
+<!-- Testing page to try out if and else statements-->
+
+
+<h1 style="text-align: center"> Tea & Coffee </h1>
+<br> <center><img src="https://uploads.disquscdn.com/images/7b7eb86cc5695faa31ee4c015f42cba1f3d9a9e719c2f020dd14b64234783741.jpg"></center>
+
+
+
+<center>
+<h2 style="text-align: center"> Which type of drink is right for you? </style> </h1>
+
+<p>How do you feel?</p>
+
+<select id="feeling" onchange="myFunction()">
+  <option value="none">
+  <option value="Pumped">Pumped
+  <option value="Drowsy">Drowsy
+ 
+</select>
+
+
+<br><br>
+<p>Strength?</p>
+
+<select id="type" onchange="myFunction()">
+  <option value="none">
+  <option value="Refreshing">Refreshing
+  <option value="Strong">Strong
+</select>
+
+<p>Here is your drink....</p>
+
+<p id="demo"></p>
+
+<script>
+function myFunction() {
+    var x = document.getElementById("feeling").value;
+    var y = document.getElementById("type").value;
+   	
+   	if (x == "Pumped" && y == "Refreshing") {
+   	 document.getElementById("demo").innerHTML = "Moroccan Mint Tea!";
+	} else if (x == "Pumped" && y == "Strong") {
+   	 document.getElementById("demo").innerHTML = "Arabica Colombia Dark Roast Coffee!";
+	} else if (x == "Drowsy" && y == "Refreshing") {
+   	 document.getElementById("demo").innerHTML = "Hawaiian Kona Coffee";
+	} else if (x == "Drowsy" && y == "Strong") {
+   	 document.getElementById("demo").innerHTML = "English Early Grey Tea.";
+	} else {
+		document.getElementById("demo").innerHTML = "Please select both fields";
+	}} 
+</script>
+
+</center>
+
+<center>
+	<script type="text/javascript">
+	
+	function budgetCheck() {
+	if (document.location1.text1.value > 4) {
```

Comment:
Why do you reference the element via `document` here, but use `getElementById` up above?

Similar to the feedback I gave Will, I would suggest caching the element and referencing parameters from the variable. It's more explicit to readers that way among other things...

```
var budget = document.getElementById('budget');

if (budget.value > 4) {
  // do something
}
```

(note you would need to add an ID to the input field)